src/config.rs

```diff
@@ -718,6 +723,47 @@ pub fn parse_config_file(buffer: &str, verbosity: u64) -> Args {
             }
             firehosen
         });
+
+    args.kinesises = sinks.get("kinesis").map(|snk| {
```

Comment:
Could you also make a test for this in the basement of this module?

Reply:
Done.